lib/config.js

```diff
@@ -1,16 +1,15 @@
 import convict from 'convict';
 import yaml from 'js-yaml';
 import pino from 'pino';
-import path, { dirname, join } from 'path';
+import path, { join } from 'path';
 import fs from 'fs';
 import os from 'os';
-import { fileURLToPath } from 'url';
 
-const __dirname = dirname(fileURLToPath(import.meta.url));
+const { pathname } = new URL("./", import.meta.url);
 
 let pack = {};
 try {
-    pack = JSON.parse(fs.readFileSync(join(__dirname, '../../package.json')));
+    pack = JSON.parse(fs.readFileSync(join(pathname, '../../../../package.json')));
```

Comment:
Is this reaching out of node_modules back to the users app folders? 
Should we maybe be using `process.cwd()`instead if that's the case?
I'm guessing this would break in an npm 7 workspaces situation where the node_modules folder is not the same folder as app.

Reply:
Good point.